compiler/qsc_doc_gen/src/generate_docs/tests.rs

```diff
@@ -211,3 +278,115 @@ fn top_index_file_generation() {
     "#]]
     .assert_eq(full_contents.as_str());
 }
+
+#[test]
+fn dependency_with_main_namespace_fully_qualified_name() {
+    let full_contents = check_doc_generation(
+        Some(("dep/Main.qs", "namespace Main { operation DependencyFunction() : Unit {} export DependencyFunction; }")),
```

Comment:
I think this should not explicitly declare the `Main` namespace as that is not the norm in Q# anymore. At least there should be a test that does the same as this test but without that explicit namespace declaration.